### PR TITLE
Updated doc to replace signalfx-json with signalfx as the endpoint type

### PR DESCRIPTION
**Pull request by harnitsignalfx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,14 +274,14 @@ the carbon server is.
         }
 ```
 
-#### signalfx-json
+#### signalfx
```
Comment (mdubbyap) [inline README.md]: fix your travis checks, now have two headings the same.

File: protocol/wavefront/wavefrontlistener.go [hunk 2]
```diff
@@ -139,7 +139,7 @@ func (listener *Listener) fromWavefrontDatapoint(line string) *datapoint.Datapoi
 	for _, p := range pieces {
 		dimPieces := strings.SplitN(p, "=", 2)
 		if len(dimPieces) == 2 {
-			dimensions[dimPieces[0]] = dimPieces[1]
+			dimensions[dimPieces[0]] = strings.Replace(dimPieces[1],"\"","",-1)
```
Comment (mdubbyap) [inline protocol/wavefront/wavefrontlistener.go]: don't put two completely different things in one pull request. 

also, this is too trivial to be functional.  i'll propose a thorough solution.

**Comment by harnitsignalfx** [comment]
All set.

**Comment by mdubbyap** [comment]
Update name of review and commit to be more specific and accurate.

**Comment by harnitsignalfx** [comment]
All set, again !

**Pull request merged**